Opio-Emmanuel-Omona/code-labs
# #155820624 Launch the detail activity after clicking list item

#### What does this PR do?
- [x] This PR launches the detail activity after clicking a list item in the main activity.

#### Description of Task to be completed?
- [x] add onClick listeners
- [x] implement intents
- [x] pass data through intents

#### How should this be manually tested?
* Clone the repo
```git clone https://github.com/Opio-Emmanuel-Omona/code-labs.git```
* Check into this branch
```git checkout ft-launch-detail-activity-155820624```
* Run the application

* Click on any item in the list

#### Relevant pivotal tracker stories
[ #155820624](https://www.pivotaltracker.com/story/show/155820624)

#### Screenshots
![image](https://res.cloudinary.com/opix/image/upload/v1553531763/launch_detail_activity.gif)
